libweb3jsonrpc/test.json

```diff
@@ -1,4 +1,5 @@
 [
+{ "name": "test_closeClient", "params": [], "order": [], "returns": false},
```

Comment:
I'm just curious; what does this "returns" field mean?

Reply:
I guess. false or true  means bool.  and "" means string. judgind on how other .json descriptions are made